src/env/setup_virt_card.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Is this yaml file used anywhere? Is it needed for setup_virt_card.sh? If yes why do we need separate yaml file, why can't it be part of the .sh script?

Reply:
not, it is not used. keep it here just for a lower number of changes in this PR :D But ok, will remove it